auto_round/__main__.py

```diff
@@ -154,6 +154,16 @@ def setup_parser():
     parser.add_argument("--fp_layers_list", default="", type=str,
                         help="List of Layers to maintain original data type")
 
+    # ======================= VLM =======================
+    parser.add_argument("--quant_vision", action='store_true',
+                        help="To determine whether the quantization should handle vision component.")
+
+    parser.add_argument("--images", default="coco", type=str,
```

Comment:
replace this arg with others 
All we need is a dataset name The string can represent either a Hugging Face dataset name or a local file path.

The dataset metadata file must be in JSON format and should include information about text and other modalities, such as images and videos.

Users have the option to provide a dictionary or a string(only for only one extra modality)supply additional information, enabling the retrieval of corresponding images, videos, or audio files.